packages/replay-next/pages/variables.css

```diff
@@ -48,7 +48,7 @@
   --primary-accent: #01acfd;
   --primary-accent-hover: #0194ff;
 
-  --secondary-accent: var(--secondary-accent);
+  --secondary-accent: #d72451;
```

Comment:
Hmm, this PR didn't get reviewed, and it seems to have set secondary accent and secondary accent hover to the same color.